src/main/resources/templates/update-book.html

```diff
@@ -6,47 +6,42 @@
-            </p>
-            <p>
-                <label for="price">Price:</label>
-                <input id="price" type="number" step="0.01" min="0" th:field="*{price}">
-                <span class="error-input" th:if="${#fields.hasErrors('price')}" th:errors="*{price}"></span>
-            </p>
-            <p><input id="submit" type="submit" value="Update Book" /> <input type="reset" value="Reset" /></p>
+                <fieldset>
+                    <label for="name">Name:</label>
+                    <span class="error-input" th:if="${#fields.hasErrors('name')}" th:errors="*{name}"></span>
+                    <input id="name" type="text" minlength="2" th:field="*{name}">
+                       
+                    <label for="description">Description:</label>
+                    <span class="error-input" th:if="${#fields.hasErrors('description')}" th:errors="*{description}"></span>
+                    <textarea id="description" name="description" minlength="2" th:field="*{description}"  rows="5"></textarea>
+            
+                    <label for="ISBN">ISBN:</label>
+                    <span class="error-input" th:if="${#fields.hasErrors('ISBN')}" th:errors="*{ISBN}"></span>
+                    <input id="ISBN" type="text" minlength="13" maxlength="13" th:field="*{ISBN}">
+                    
+                    <label for="picture">Picture:</label>
+                    <span class="error-input" th:if="${#fields.hasErrors('picture')}" th:errors="*{picture}"></span>
+                    <input id="picture" type="file" th:field="*{picture}">
+                
+                    <label for="author">Author:</label>
+                    <span class="error-input" th:if="${#fields.hasErrors('author')}" th:errors="*{author}"></span>
+                    <input id="author" type="text" minlength="2" th:field="*{author}">
+                
+                    <label for="publisher">Publisher:</label>
+                    <span class="error-input" th:if="${#fields.hasErrors('publisher')}" th:errors="*{publisher}"></span>
+                    <input id="publisher" type="text" minlength="2" th:field="*{publisher}">
+                
+                    <label for="inventory">Inventory:</label>
+                    <span class="error-input" th:if="${#fields.hasErrors('inventory')}" th:errors="*{inventory}"></span>
+                    <input id="inventory" type="number" min="0" th:field="*{inventory}">
+                    
+                    <label for="price">Price:</label>
+                    <span class="error-input" th:if="${#fields.hasErrors('price')}" th:errors="*{price}"></span>
+                    <input id="price" type="number" step="0.01" min="0" th:field="*{price}">
+                    
+                    <input id="submit" type="submit" value="Update Book" />
+                    <input type="reset" value="Reset" />
+                </fieldset>
```

Comment:
In a later refactoring PR we might want to just merge create-book.html and update-book.html into the same file and compare check a query param or something so that this fieldset code isn't duplicated in two files, seems like it could get annoying having to maintain two different files if we change a single field name or anything on here

Reply:
I agree, I belive @Hajar1 wanted to do that.

Reply:
can we please confirm there is an issue created for this specific refactor or create one if we don't have one yet

Reply:
Created an issue for this at #73